src/Strategy/Conda/EnvironmentYml.hs

```diff
@@ -73,9 +87,36 @@ getCondaDepFromText rcd =
             Nothing -> ""
       pure (aVal <> bVal)
 
+newtype PackageManager = PackageManager Text
+  deriving newtype (Eq, Ord, Show, FromJSON)
+
+data CondaPkg
+  = Pkg Text
+  | PipPkg [Req]
+  deriving (Eq, Ord, Show)
+
+instance FromJSON CondaPkg where
+  parseJSON cd = (Pkg <$> parseJSON cd) <|> withObject "CondaPipPkg" parseOtherPkgManager cd
+    where
+      parseOtherPkgManager :: Object -> Parser CondaPkg
+      parseOtherPkgManager (KeyMap.toList -> [("pip", deps)]) = trace "Pip package!" $ withArray "Pip Packages" parseReqs deps
+      parseOtherPkgManager obj = fail ("Expected object with a single key (\"pip\") and list of deps. Got: " <> show obj)
+
+      parseReqs :: Array -> Parser CondaPkg
```

Comment:
In the Slack thread and in this PR's description, you said that this would fail for URLs. If this is true, should we add a comment giving an example of what fails and noting that it does not work yet?

Reply:
Good idea, I've added some text about this.